lua/cfc_random_spawn/module/sv_player_spawning.lua

```diff
@@ -42,12 +97,17 @@ function CFCRandomSpawn.getOptimalSpawnPosition()
     return spawnPoinTbl.spawnPos, spawnPoinTbl.spawnAngle
 end
 
-function CFCRandomSpawn.updateSpawnPointRankings()
+function CFCRandomSpawn.updateSpawnPointRankings( ply )
     local playerIDSFromSpawns = {}
+    local measurablePlayers = getMeasurablePlayers( ply )
+
+    popularPoint = getPopularPoint( measurablePlayers )
+
+    bestSpawns = getNearestSpawns( popularPoint, customSpawnsForMap, nearSpawnpointsMinCount, nearSpawnpointsMinDistance )
```

Comment:
You're constantly inputting `nearSpawnpointsMinDistance` here eventhough it never changes, might wanna just use that in the function itself

Reply:
Same for `nearSpawnpointsMinCount`